hack/deploy.sh

```diff
@@ -0,0 +1,205 @@
+	pkgmanifest=`oc get packagemanifest "${OPERATOR_PKG_NAME}" -n openshift-marketplace -o jsonpath='{.status.catalogSource}' 2> /dev/null`
+	if [ ! "$pkgmanifest" = "${OPERATOR_PKG_NAME}" ]; then
+		if [ "${n}" = "60" ]; then
+			error 'timed out waiting'
+			info 'catalog source pod yaml:'
+			oc -n openshift-marketplace get pods -l=olm.catalogSource="${OPERATOR_PKG_NAME}"
+			exit 1
+		fi
+		sleep 10
+		continue
+	fi
+	info 'catalog source installed and available'
+	break
+done
+
+info 'installing the operator'
+
+yq e -i '.spec.targetNamespaces[0] = env(NAMESPACE)' "${OG_PATH}"
+oc apply -n "${NAMESPACE}" -f "${OG_PATH}"
+
+yq e -i '
+	.spec.channel = "test" |
+	.spec.startingCSV = env(OPERATOR_PKG_NAME) |
+	.spec.name = env(OPERATOR_PKG_NAME) |
+	.spec.source = env(OPERATOR_PKG_NAME)
+' "${SUBSCRIPTION_PATH}"
+oc apply -n "${NAMESPACE}" -f "${SUBSCRIPTION_PATH}"
+
+info 'waiting for CSV...'
+
+for n in {1..60}; do
+	phase=`oc get csv "${OPERATOR_PKG_NAME}" -n "${NAMESPACE}" -o jsonpath='{.status.phase}' 2> /dev/null`
+	if [ ! "$phase" = 'Succeeded' ]; then
+		if [ "${n}" = "60" ]; then
+			error 'timed out waiting'
+			info 'csv contents:'
+			oc get csv "${OPERATOR_PKG_NAME}" -n "${NAMESPACE}" -o yaml
+			exit 1
+		fi
+		sleep 10
+		continue
+	fi
+	info 'CSV successfully installed'
+	break
+done
+
+info 'deploy Quay'
+oc apply -n "${NAMESPACE}" -f "${QUAY_SAMPLE_PATH}"
+
+# wait for quay deployment to come up. the next checks will fail if the
+# resources don't exist.
+
+info 'waiting for quay deployment...'
+
+for n in {1..60}; do
+	deploy=`oc get deploy skynet-quay-app -n "${NAMESPACE}" -o jsonpath='{.metadata.name}' 2> /dev/null`
+	if [ ! "$deploy" = 'skynet-quay-app' ]; then
+		if [ "${n}" = "60" ]; then
+			error 'timed out waiting'
+			exit 1
+		fi
+		sleep 10
+		continue
+	fi
+	info 'quay deployment created'
+	break
+done
+
+info 'waiting for pods to come up...'
+
+# the order below matters!
+# the mirror is the last to come up, so after the dependencies are ready
+# we wait for the mirror to be ready, then we check the quay app as it's
+# more likely to have stabilized after the mirror is up and running.
+oc -n "${NAMESPACE}" wait pods -l=quay-component=postgres --for=condition=Ready --timeout="${WAIT_TIMEOUT}"
+oc -n "${NAMESPACE}" wait pods -l=quay-component=redis --for=condition=Ready --timeout="${WAIT_TIMEOUT}"
+oc -n "${NAMESPACE}" wait pods -l=quay-component=quay-config-editor --for=condition=Ready --timeout="${WAIT_TIMEOUT}"
+oc -n "${NAMESPACE}" wait pods -l=quay-component=quay-mirror --for=condition=Ready --timeout="${WAIT_TIMEOUT}"
+oc -n "${NAMESPACE}" wait pods -l=quay-component=quay-app --for=condition=Ready --timeout="${WAIT_TIMEOUT}"
+
+
+# manually check quay's health to ensure it can successfuly connect to its components.
+endpoint=`oc get quayregistry skynet -o jsonpath='{.status.registryEndpoint}'`
+result=`curl -s -k "${endpoint}"/health/instance`
+
+if [ ! `echo "${result}"|jq '.status_code'` = "200" ]; then
```

Comment:
```suggestion
if [ "$(echo "$result" | jq '.status_code')" != "200" ]; then
```

Reply:
Good catch!
I've applied the change to this and all following conditions, so I won't commit the suggestion via UI.